src/manifest.json

```diff
@@ -19,7 +19,7 @@
   "devtools_page": "devtools.html",
   "web_accessible_resources": [{ "resources": ["inpage.js"], "matches": ["*://*/*"] }],
   "host_permissions": ["*://*/*"],
-  "permissions": ["storage"],
+  "permissions": ["storage", "tabs"],
```

Comment:
Yeah ok then we need another implementation. Let me think about it, but leaning towards subscribe to event listeners on local storage

Reply:
yeah could also use Broadcast channel. benefit of that is the broadcaster receives its own message, so if we enable account switching from a tab, it would still work